rules/create.go

```diff
@@ -142,15 +142,27 @@ func getSnakes(req *pb.CreateRequest) ([]*pb.Snake, error) {
 	return snakes, nil
 }
 
-func generateFood(req *pb.CreateRequest, snakes []*pb.Snake) ([]*pb.Point, error) {
-	food := []*pb.Point{}
-
-	for i := int32(0); i < req.Food; i++ {
-		p := getUnoccupiedPoint(req.Width, req.Height, food, snakes)
-		if p != nil {
-			food = append(food, p)
-		}
+func randomNeighbour(coord *pb.Point) *pb.Point {
+	directions := []Coords{
+		{1,0},
+		{-1,0},
+		{0,1},
+		{0,-1},
 	}
+	i := rand.Int() % len(directions)
```

Comment:
```suggestion
	i := rand.Intn(len(directions))
```

Assuming this is available in go 1.11 (i think it should be)